Syntaxes/debian-control.sublime-syntax

```diff
@@ -9,45 +9,59 @@ scope: source.debian.control
 variables:
   # https://www.debian.org/doc/debian-policy/ch-controlfields.html#source-package-control-files-debian-control
   allowed_fields: |-
-    (?i)^(
-      |Source|Maintainer|Uploaders|Section|Priority|Build-Depends(-Arch|-Indep)?|
-      |Build-Conflicts(-Arch|-Indep)?|Standards-Version|Homepage|
-      |Vcs-(Browser|Arch|Bzr|Cvs|Darcs|Git|Hg|Mtn|Svn)|Testsuite|Rules-Requires-Root|
-      |Package|Architecture|Essential|Depends|Pre-Depends|Recommends|Suggests|Breaks|
-      |Conflicts|Provides|Replaces|Enhances|Description|Homepage|Built-Using|Package-Type|
-    )
-
-  # https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#substvar
-  allowed_substvar: |-
-    (
-      |\${misc:Depends}|\${misc:Pre-Depends}|\${shlibs:Depends}|\${python:Depends}|
-      |\${python3:Depends}|\${perl:Depends}|\${ruby:Depends}|
+    (?xi:
+       Source|Maintainer|Uploaders|Section|Priority|Build-Depends(?:-Arch|-Indep)?
+      |Build-Conflicts(?:-Arch|-Indep)?|Standards-Version|Homepage
+      |Vcs-(?:Browser|Arch|Bzr|Cvs|Darcs|Git|Hg|Mtn|Svn)|Testsuite|Rules-Requires-Root
+      |Package|Architecture|Essential|Depends|Pre-Depends|Recommends|Suggests|Breaks
+      |Conflicts|Provides|Replaces|Enhances|Description|Homepage|Built-Using|Package-Type
     )
 
 contexts:
   main:
-    - include: comments
-    - include: email-address
+    - include: debian-common.sublime-syntax#comments
     - include: fields
-    - include: substvar
 
-  comments:
-    - match: "#"
-      scope: punctuation.definition.comment.debian
+  fields:
+    - match: ^({{allowed_fields}})(:)?
+      captures:
+        1: meta.mapping.key.debian keyword.control.debian.control
+        2: punctuation.separator.mapping.key-value.debian.control
       push:
-        - meta_scope: comment.line.debian
-        - match: \n
+        - meta_content_scope: meta.mapping.value.debian.control
+        - match: ^(?=\S)
           pop: true
+        - include: substvar
+        - include: semver-flag
+        - include: debian-common.sublime-syntax#email-address
+        - include: debian-common.sublime-syntax#url
+        - match: \,|\|
+          scope: punctuation.separator.sequence.debian.control
 
-  email-address:
-    - match: <.+@.+[.].+>|<.+?at.+?dot.+?>
-      scope: variable.language.debian
-
-  fields:
-    - match: '{{allowed_fields}}'
-      scope: keyword.control.debian
-
+  # https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#substvar
   substvar:
-    - match: '{{allowed_substvar}}'
-      scope: variable.function.debian
+    - match: |-
```

Comment:
I'd keep this in `variables` for consistency's sake. I just think it looks cleaner than having a multi regex in the context